src/commands/deploy.ts

```diff
@@ -287,8 +287,27 @@ export async function deployFrontend(): Promise<string> {
   const configuration = await getProjectConfiguration();
 
   if (configuration.frontend) {
+
+    // check if the build folder exists
+    if (!await fileExists(configuration.frontend.path)) {
+      throw new Error(`The build folder does not exist. Please run the build command first or add a preFrontendDeploy script in the genezio.yaml file`)
+    }
+
+    // hcheck if the build folder is empty
+    if (await isDirectoryEmpty(configuration.frontend.path)) {
+      throw new Error(`The build folder is empty. Please run the build command first or add a preFrontendDeploy script in the genezio.yaml file`)
+    }
+
+    // check if there are any .html files in the build folder
+    if (!await ifDirectoryContainsHtmlFiles(configuration.frontend.path)) {
```

Comment:
it's weird to start the name of a function with `if`. If it returns a boolean value, you can name it `directoryContainsHtmlFiles`.